.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,69 @@
+name: "Benchmarks"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
```

Comment:
Can we make this a little more optimal and make the benchmark part of the existing tests here https://github.com/appwrite/appwrite/blob/main/.github/workflows/tests.yml#L41-L47

That way we already have the image built and saved in cache. 

We can add a new job called `benchmark` that uses the cached image from the `setup` job.

Reply:
Done